apps/backoffice-v2/src/common/components/organisms/Filters/context/utils/create-filter-values-map.ts

```diff
@@ -0,0 +1,24 @@
+import { TFiltersValuesMap } from '@/common/components/organisms/Filters/context/types';
+import { IFilterDefinition, IFilterValue } from '@/common/components/organisms/Filters/interfaces';
+
+export const createFilterValuesMap = ({
```

Comment:
What's this function for?

Reply:
Creating map so values could be easily accessed by filter inputs.